src/change_stream.ts

```diff
@@ -357,7 +363,7 @@ export class ChangeStream<TSchema extends Document = Document> extends TypedEven
    */
   stream(options?: CursorStreamOptions): Readable {
     this.streamOptions = options;
-    if (!this.cursor) throw new MongoDriverError(NO_CURSOR_ERROR);
+    if (!this.cursor) throw new MongoInvalidArgumentError(NO_CURSOR_ERROR);
```

Comment:
Happy to merge this for now, but I think this should ultimately end up as `MongoChangeStreamError` which is covered by [NODE-3404](https://jira.mongodb.org/browse/NODE-3404).

Reply:
I think this should be changed back to `MongoDriverError`, at least until `MongoChangeStreamError` is introduced. A change stream not having an internal cursor would not be a result of invalid user input.